src/index.js

```diff
@@ -35,15 +35,23 @@ const processArgs = args => {
   }
 }
 
+const makeCardName = (namespace, opts) => {
+    //TODO: Make this better
+    return namespace + "__" + opts.doc.split('\n')[0];
```

Comment:
@elliottconduce @RonAtConduce Are there any rules on the card names?  Like spaces failing in urls

Reply:
We had issues with namespace names having spaces.  Since you're effectively replacing the namespace with this name, I expect we'll run into the same issues.  I wonder if instead of using the markdown string if you could just get away with using the "key" field on the CardImpl component.  It's ugly and would make order matter, but it would at least be unique.  Another option is to actually change the API and add a name string argument for cards.  That's probably nicer in the long run

Reply:
Another option is to replace the spaces with dashes or underscores.

I had thought about making a different class or method as well, as well so we would run different code to create single page cards, but I didn't want to have to make changes to the card files.